django/contrib/admin/templates/admin/index.html

```diff
@@ -11,10 +11,17 @@
 
 {% block nav-sidebar %}{% endblock %}
 
+{% block content_title %}
+  <div class="admin-index-title">
+    {% if title %}<h1>{{ title }}</h1>{% endif %}
```

Comment:
Could you replace this with `block.super`?
```suggestion
    {{ block.super }}
```